tests/test.rs

```diff
@@ -95,14 +95,17 @@ fn test_remove_prop(mut string: String, index: usize) -> bool {
 fn test_all_prop(mut string: String, commands: Vec<Command>) -> bool {
     let mut fast_string = FastString::new(string.as_str());
     let mut clones = Vec::new();
+
     for command in commands.iter() {
+        // :thumbsup:
         let result = match command {
             Command::Push { ch } => push_command(&mut string, &mut fast_string, *ch),
             Command::PushStr { add } => {
                 push_str_command(&mut string, &mut fast_string, add.as_str())
             }
             Command::Remove { idx } => remove_command(&mut string, &mut fast_string, *idx),
             Command::Clone => {
+                // Do we want to assert some props about clones?
```

Comment:
I would like to make sure that after cloning the string it remains in the same state and after that it works correctly